colcon_bundle/verb/bundle.py

```diff
@@ -198,14 +189,13 @@ def _manage_dependencies(self, context,
                             arcname=os.path.join(
                                 name, os.path.basename(file_path)))
 
-        if dependencies_changed:
-            update_symlinks(staging_path)
-            # TODO: Update pkgconfig files?
-            update_shebang(staging_path)
-            # TODO: Move this to colcon-ros-bundle
-            rewrite_catkin_package_path(staging_path)
+        update_symlinks(staging_path)
+        # TODO: Update pkgconfig files?
+        update_shebang(staging_path)
+        # TODO: Move this to colcon-ros-bundle
+        rewrite_catkin_package_path(staging_path)
 
-        return dependencies_changed
+        return True
```

Comment:
Should this always be true?

Reply:
Yeah, in `bundle.py`, it returns False only when it detects same local dependency or same dependencies in the dependency graph. Otherwise, it will fall into the source compression/symlink update part.